locale/en.json

```diff
@@ -951,7 +953,24 @@
   "REVOKED_COLLAB_EXISTS": "Could not create collaboration - a revoked collaboration between those users already exists",
   "COLLAB_REVOKE_ERROR_SUPPLEMENTAL": "Note that collaborations cannot be revoked unless they are mutually approved.",
   "PENDING_SIGHTINGS": "Pending Sightings",
-  "SENT_YOU_A_COLLABORATION_REQUEST": "{senderName} sent you a collaboration request",
+  "SENT_YOU_A_COLLABORATION_REQUEST": "{userName} sent you a collaboration request",
   "A_COLLABORATION_WAS_CREATED_ON_YOUR_BEHALF": "A collaboration was created on your behalf",
-  "TIME_SINCE": "{timeSince} ago"
+  "A_COLLABORATION_WAS_CREATED_ON_YOUR_BEHALF_MORE_DETAILED": "A collaboration was created on your behalf between {user1Name} and {user2Name}",
+  "TIME_SINCE": "{timeSince} ago",
+  "COLLABORATION_APPROVED": "Your collaboration with {userName} has been approved",
+  "COLLABORATION_APPROVED_TITLE": "Collaboration approved",
+  "COLLABORATION_REVOKE_BRIEF": "{userName} revoked your collaboration request",
+  "EDIT_COLLABORATION_APPROVED": "{userName} approved your collaboration edit request",
+  "EDIT_COLLABORATION_REVOKED": "{userName} revoked edit privileges for your collaboration with them",
+  "COLLABORATION_REVOKED_BY_MANAGER": "A collaboration was revoked by a user manager",
+  "INDIVIDUAL_MERGE_REQUEST_MESSAGE": "{userName} requested that an individual in your data set be merged with another individual in the database",
```

Comment:
For this one is there any way to access the names of the individuals to be merged? I was thinking something like `Jake Holland wants to merge "Stegosaurus" with "Big Beefy".`